extensions/amp-smartlinks/0.1/constants.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * Copyright 2018 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
```

Comment:
you can export the whole config obj
```
export {
  KEY: 'val',
};
```

Reply:
I went with a mixture of the old style and your suggestion. Does this look good to you? https://github.com/hellonarrativ/amphtml/pull/3/files#diff-ab226ee6b19deb3b34ee6dcb98e1ac02R20